app/client/pages/my/spaces/[space_id]/resource/[resource].js

```diff
@@ -72,11 +77,14 @@ export default function resource() {
                         <span>{new Date(data?.resource?.createdAt).toLocaleDateString()}</span>
 
                     </div>
+                    <div className={styles.resourceDetailHeader}> 
```

Comment:
This needs a left margin

Reply:
I think nurlan is implementing this page. I just added a button for note creation.There might be merging issues.